ansible/roles/mongodb/templates/mongod.conf.j2

```diff
@@ -0,0 +1,16 @@
+storage:
+  dbPath: {{ mongo_data_path | default("/var/lib/mongodb") }}
```

Comment:
I'd suggest putting this default into the actual `defaults/main.yml` file, if we're trying to keep a maintainable role.